update_examples.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
```

Comment:
nice! Consider adding to the repo a `development.md` file to introduce the script to sdk developers, describing how and when to use it. example: https://github.com/numaproj/numaflow/blob/main/docs/development/development.md